README.asciidoc

```diff
@@ -369,6 +369,19 @@ command names:
 ------------------------------------------------------------------------------------------
 
 
+[[Scripting]]
+Hints on scripted usage
+-----------------------
+If you want to run image-bootstrap from a shell script and capture its output
+to STDOUT/STDERR in a log file, be sure to run image-bootstrap in unbuffered
+mode, e.g.:
+
+`python -u image-bootstrap [params] 2>&1 | tee my.log`
```

Comment:
I wonder what the `stdbuf(1)` version of that command would be. Probably not of more value here but in case you figure it out, please let me know.

Reply:
Its man page says also:

>  NOTE: If COMMAND adjusts the buffering of its standard streams ('tee'  does
       for  example)  then  that  will override corresponding changes by 'stdbuf'.

Since python obviously does its own thing, I'm not sure if this could be trusted. Never used stdbuf before and probably never will :)